src/components/EditableField.jsx

```diff
@@ -31,7 +31,7 @@ const EditableField = (props) => {
   }, [value]);
 
   const deleteEditableField = (e) => {
-    props.onDelete(e);
+    props.onDelete(e, editedValue);
```

Comment:
Required for https://github.com/biomage-ltd/ui/blob/a2ee1563a58a2a00f79d411ef2318b626ecf3951/src/components/data-management/ProjectDetails.jsx#L266